web/README.md

```diff
@@ -1,26 +1,33 @@
 # Web
+[comment]: <> (Double check Americanised vs Anglicised version of spellings throughout all documentation as well.)
+[comment]: <> (Double check consistency with "front-end" vs "front end" throughout documentation.)
+This is the main front-end website project. This project will consume the output of the front-end components project via NPM.
 
-This is the main front-end website project. This project will consume the output of the front end components project via NPM.
-
-This is an MVC web app written in C#. It's main purpose is to provide proxy authentication/authorization services, along with any static pages that need serving
+This is an MVC web app written in C#. It's main purpose is to provide proxy authentication/authorisation services, along with any static pages that need serving
 
 [![Build Status](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_apis/build/status%2FWeb%20CICD?branchName=main)](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_build/latest?definitionId=2600&branchName=main)
 
 ## Getting started
 
 ### Run the Web App locally
-
+[comment]: <> (Should we specify our dependencies here? Or link to the document on how to set up the dependencies?)
```

Comment:
Any non nuget or npm should be documented, but we don't have any at the moment.

Reply:
Ok, something to track. Could be added as a commented "TODO"?

Reply:
We may never have any, and then in which case the TODO would be redundant.